apmpackage/apm/data_stream/traces/manifest.yml

```diff
@@ -16,3 +16,9 @@ elasticsearch:
             mapping:
               type: scaled_float
               scaling_factor: 1000000
+      properties:
+        timestamp:
+          properties:
+            us:
+              type: long
+              index: false
```

Comment:
I think we should instead update `fields/fields.yml`:

https://github.com/elastic/apm-server/blob/ff5e1792298a52fffef1e5079917942b6b364c94/apmpackage/apm/data_stream/traces/fields/fields.yml#L248-L251

IIANM you can just add `index: false` to the field definition there.

Reply:
Thanks for the tip! :bow: 

I've updated the poc to use the fields mentioned in the related issue

Reply:
Thanks, looks good :)